code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -379,6 +409,15 @@
 			M.reagents.remove_reagent(R.id,1)
 	..()
 
+/datum/reagent/medicine/charcoal/on_reagent_conflict(mob/living/M, datum/reagent/CR)
+	if(!..())
+		return FALSE
+	switch(CR.id)
+		if("styptic_powder", "silver_sulfadiazine", "synthflesh", "bicaridine", "kelotane", "tricordrazine")
```

Comment:
couldn't it just check if its in the conflicting reagents? you already made a list for this.